mne/surface.py

```diff
@@ -1889,14 +1901,14 @@ def warp_montage_volume(montage, base_image, reg_affine, sdr_morph,
     _warn_missing_chs(montage, image_to, after_warp=True)
 
     # recover the contact positions as the center of mass
-    warped_data = np.asanyarray(image_to.dataobj)
+    warped_data = np.array(image_to.dataobj)
     for val, ch_coord in enumerate(ch_coords, 1):
-        ch_coord[:] = np.asanyarray(
+        ch_coord[:] = np.array(
             np.where(warped_data == val), float).mean(axis=1)
```

Comment:
Same. In fact you shouldn't even need the `np.array` at all:
```suggestion
        ch_coord[:] = (np.where(warped_data == val), float).mean(axis=1)
```